### ISSUE TITLE
[Bug Report] bootstrap module creates a storage account but no fileshares

### ISSUE THREAD

**Issue by vinnyoflegend** [PaloAltoNetworks/terraform-azurerm-swfw-modules#102]
### Describe the bug

Tested with Terraform 1.9.6 and 1.9.7 on Windows/AMD64 platform, I copied https://github.com/PaloAltoNetworks/terraform-azurerm-swfw-modules/blob/main/examples/gwlb_with_vmseries/example.tfvars to terraform.tfvars and modified some of the variables for my needs (prefix, region, names, etc), copy/renamed "init-cfg.sample.txt" to init-cfg.txt. I also tried adding an authcodes file and mapping it to licenses/authcodes in the tfvars. (So my line reads as `static_files = { "files/init-cfg.txt" = "config/init-cfg.txt", "files/authcodes" = "license/authcodes" }`

Terraform plan and apply succeed with no warnings/issues but the fileshares for bootstrapping are not created. I only see the storageaccount itself is provisioned. No bootstrap files appear to be processed.

I tried this on two Azure subscriptions, including a new personal one to rule out any governance/enterprise controls in my workplace. The outcome was the same on both.

### Module Version

3.1.0 

### Terraform version

1.9.7

### Expected behavior

According to the documentation, the module should create a fileshare with structure for the bootstrap folders/files.

### Current behavior

The module only creates an Azure storage account in the resource group.

**Comment by acelebanski** [comment]
Hello @vinnyoflegend, thanks for raising this issue. I already identified this problem and it will be fixed with PR #98. 

The workaround is to move `bootstrap_package` from `vmseries_universal` map directly to `vmseries` -> `virtual_machine` map. Like this:

```hcl
vmseries = {
  "fw-1" = {
    name     = "firewall01"
    vnet_key = "transit"
    virtual_machine = {
      zone = 1
      bootstrap_package = {
        bootstrap_storage_key  = "bootstrap"
        static_files           = { "files/init-cfg.txt" = "config/init-cfg.txt" }
        bootstrap_xml_template = "templates/bootstrap-gwlb.tftpl"
        data_snet_key          = "data"
      }
    }
  (...)
  }
}
```

**Comment by vinnyoflegend** [comment]
That's great to hear, I'll use the workaround for now. Thanks again for the quick response!
